app/scss/_eventsDetails.scss

```diff
@@ -1,294 +1,182 @@
-
-
-
-        }
-
-      }
-
-      .event-sidebar {
-        display: inline-block;
-        width: 250px;
-        text-align: left;
-
-        .img-container{
-          position: relative;
-          width: 250px;
-          height: 250px;
-          display: inline-block;
-          margin-bottom: 10px;
-          img {
-            position: absolute;
-            width: 100%;
-            height: auto;
-          }
-          .border{
-            position: absolute;
-            width: 125px;
-            height: 125px;
-            bottom: -8px;
-            right: -8px;
-            background-color: blue;
-          }
-        }
-        div{
-          display: inline-block;
-          p{
-            font-size: 14px;
-          }
-          .name{
-            font-weight: bolder;
-          }
-
-          .description>p{
-            color: gray;
-            font-size: 14px;
-          }
-        }
+      margin-top: 15px;
+      margin-left: 25%;
+      margin-right: 25%;
+      border-bottom: solid $orange 3px;
+      box-sizing: border-box;
+    }
+  }
 
-        @media screen and (max-width: 1200px)
-        {
-          width: 900px;
-          margin-top: 100px;
-          margin-left: auto;
-          margin-right: auto;
-          .img-container {
-            margin-left: 50px;
-          }
+  &--description{
+    width: 65%;
+    margin: 0;
+    box-sizing: border-box;
+    text-align: left;
+  }
+  &--title{
+    font-weight: 800;
+    font-size: 20px;
+    margin-right: 50px;
+  }
+  &--author{
+    color: gray;
+    margin-top: 10px;
+    margin-bottom: 20px;
+  }
+  &--toggle{
+    color: gray;
+    text-decoration: underline;
 
-          .description-container{
-            display: inline-block;
-            width: 300px;
-            margin-left: 40px;
-          }
-        }
+  }
+  &--toggle:hover{
+    cursor: pointer;
+  }
+  &--more{
+    position: relative;
+    text-align: justify;
+    display: flex;
+    height: 0;
+    overflow: hidden;
+    color: gray;
+    font-size: 16px;
+    strong{
+      color: black;
+    }
+  }
 
+  &--about{
+    padding: 10px;
+    width: 50%;
+  }
+  &--learn{
+    padding: 10px;
+    width: 50%;
+    text-align: left;
+    ul{
+      list-style-type: none;
+    }
+    li{
+      position: relative;
+      margin-bottom: 10px;
+    }
+    li::before{
+      content: "+";
+      display: inline-block;
+      position: absolute;
+      left: -25px;
+
+      color: orange;
+      font-size: 20px;
+      font-weight: 700;
+    }
+  }
+  &--photo{
+    padding: 5px;
+    width: 15%;
+    margin: 0;
+    box-sizing: border-box;
+    position: relative;
 
-      }
+    img{
 
+      width: 100%;
+      height: auto;
+      border-radius: 50%;
+      -webkit-box-shadow: 3px 3px 5px 0px rgba(120,120,120,1);
+      -moz-box-shadow: 3px 3px 5px 0px rgba(120,120,120,1);
+      box-shadow: 3px 3px 5px 0px rgba(120,120,120,1);
     }
-
   }
-
 }
 
+@media screen and (max-width: 960px) {
```

Comment:
Tutaj zostawił bym jeszcze ten grid który jest, tylko niech tytuł się przerzuca do nowych linijek aż do widoku mobilnego powinno to wyglądać okej (tylko już od razu przerzucić w szczegółowych informacjach sekcję 'dowiesz się' pod spód. Dopiero przy ok 520 dać godzinę na górę, tę kreskę nie jako podkreślenie, ale tak jak jest w sekcji o wydarzeniu po prawej zdjęcie a pod spodem tekst i więcej - tak jak teraz